src/sr_od/gui/interface/sim_uni/sim_uni_setting_interface.py

```
@@ -24,28 +32,82 @@ def __init__(self, ctx: SrContext, parent=None):
+    def _on_auto_simulated_universe_settings_clicked(self) -> None:
+        """
+            启动 Auto_Simulated_Universe
+        """
+        work_dir = os_utils.get_work_dir()
+        plugin_path = os.path.join(work_dir, *['plugins', 'Auto_Simulated_Universe'])
+        command = [self.ctx.python_service.env_config.python_path]
+        args = os.path.join(plugin_path, 'gui.py')
+        script_working_directory = plugin_path
+        if args and args.strip():
+            command.extend(args.split())
+        subprocess.Popen(command, cwd=script_working_directory)
```

Ruff (0.13.1)

`subprocess` call: check for execution of untrusted input

(S603)